### ISSUE TITLE
y_test.csv appears corrupted or incomplete (only ~20 rows, missing expected data)

### ISSUE THREAD

**Issue by pratham-aggr** [iharp-institute/iHARP-ML-Challenge-2#4]
The provided y_test.csv file appears to be corrupted or incomplete. In the cloned repository, it is located at Ingestion_Program/Reference data/y_test.csv, but the file contains only label-like values similar to prediction outputs and does not include any input features. Additionally, the file has only around 20 rows but it should be around 181,390 rows to match the full test set. This mismatch makes the file unusable for validation or evaluation. Could you please verify whether the uploaded y_test.csv is correct and re-upload or regenerate the intended version if necessary, or clarify if this structure is expected?

**Comment by josephine-247** [comment]
Hi, thanks for your inquiry.
The y_test file provided on GitHub is a sample to give participants insight into the structure. However, the ingestion program on Codabench utilizes the complete y_test file.
Please note the following:
- The y_test consists of all 12 stations (9 for training and 3 for testing). 
- According to the y_test sample provided, there are approximately 181k rows for the complete file, where each row value of 1 or 0 corresponds to a single date value.

**Comment by pratham-aggr** [comment]
Perfect, thanks for clarifying!

**Comment by josenino95** [comment]
If the y_test that Codabench uses includes all 12 stations (even those the model used for training), is it safe to assume the Codabench leaderboard and metrics calculated there are not a good indicator of how well my model could perform on the unseen data? So basically the leaderboard is useless?

**Comment by josephine-247** [comment]
While the leaderboard provides context about a model's performance on seen data, the ultimate evaluation will be based on unseen data, given the challenge focuses on out of distribution/domain. 
This would hence determine if a model is generalizable or overfitted.
